Cargo.toml

```diff
@@ -19,6 +19,7 @@ fuzztarget = ["secp256k1/fuzztarget", "bitcoin_hashes/fuzztarget"]
 serde-decimal = ["use-serde", "strason"]
 unstable = []
 use-serde = ["serde", "bitcoin_hashes/serde"]
+secp = ["secp256k1"]
```

Comment:
I think there's no need to rename the feature like this, since none of our dependencies transitively depend on rust-secp.

Reply:
Is `use-secp256k1` a better name? I tried first `secp256k1` but features' names must be different from dependencies' names.

Reply:
Got it, like bitcoinconsensus

Reply:
Actually, every optional dependency is automatically a feature name. It is a pretty confusing feature of Cargo, but that is what it is :D